### PR TITLE
Destination BigQuery: bump version (follow-up)

### PR DESCRIPTION
**Pull request by cynthiaxyin**
Follow-up to airbytehq/airbyte#32326 - we didn't bump BigQuery in that PR due to CI issues. So performing the bump here.

### CODE REVIEW

File: docs/integrations/destinations/bigquery.md [hunk 1]
```diff
@@ -139,11 +139,12 @@ Now that you have set up the BigQuery destination connector, check out the follo
...
-| 2.3.18  | 2023-12-04 | [33084](https://github.com/airbytehq/airbyte/pull/33084)   | T&D SQL statements moved to debug log level                                                                                                                     |
-| 2.3.17  | 2023-12-04 | [33078](https://github.com/airbytehq/airbyte/pull/33078)   | Further increase gcs COPY timeout                                                                                                                               |
+|:--------|:-----------|:-----------------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 2.3.19  | 2023-12-07 | [\#32326](https://github.com/airbytehq/airbyte/pull/32326) | Code nit fix                                                                                                                                                    |
```
Comment (evantahler) [inline docs/integrations/destinations/bigquery.md:143]: There's probably a better message we can say to the public.  Maybe "Updating common T&D interfaces"?

**Review by evantahler — APPROVED** [review]
👍, assuming tests pass

**Comment by gisripa** [comment]
> 👍, assuming tests pass

One test failed with read timed out. On that note we should probably retry few times than give up on first attempt.

**Comment by cynthiaxyin** [comment]
@gisripa good suggestion, I opened #33265 to track this separately from this PR

**Pull request merged**